### DISCUSSION TITLE
No Deb an RPM Packages on the Repository URL

### DISCUSSION THREAD

**Discussion by 53845714nF** [aquasecurity/trivy#6762] [Q&A] [answered]
### Description

In the docs are Repository listed: 
  - https://aquasecurity.github.io/trivy-repo/deb
  - https://aquasecurity.github.io/trivy-repo/rpm/releases/\$basearch/
But they are empty. Maybe the Docs are outdated. 

### Link

https://aquasecurity.github.io/trivy/v0.51/getting-started/installation/

### Suggestions

New URLs in the Docs or Packages to these URLs.

**Comment by DmitriyLewen** [comment, marked as answer]
Hello @53845714nF 
Thanks for your report!

> https://aquasecurity.github.io/trivy-repo/deb

We already merge #6636 to add `generic` dir for `deb` packages.
Next release will be include this dir.

Before that you can use old command (or release name) in `echo "deb [signed-by=/usr/share/keyrings/trivy.gpg] https://aquasecurity.github.io/trivy-repo/deb $(lsb_release -sc) main" | sudo tee -a /etc/apt/sources.list.d/trivy.list`

> https://aquasecurity.github.io/trivy-repo/rpm/releases/\$basearch/

What is `RELEASE_VERSION` are you use?

Regards, Dmitriy

**Reply by DmitriyLewen** [reply]
You already can install `v0.51.4` deb package using instruction from docs.

**Comment by 53845714nF** [comment]
Thank you very much, 

works for me.

I hadn't tested the RPM, I just thought it wouldn't work because the URL also returned a 404. 
But it was probably \$basearch/ which could not resolve the URL.

Best regards 
Sebastian
